packages/shared/src/hooks/useActions.ts

```diff
@@ -36,7 +37,7 @@ export const useActions = (): UseActions => {
 
       return [...current, ...filtered];
     },
-    { enabled: !!user },
+    { enabled: !!user, ...disabledRefetch },
```

Comment:
I think this breaks the initial flow when you create a first squad and get redirect to squad page. The checklist does not have "create squad" as completed.

Reply:
Let me try that flow quick, Don't we manually set that action FE side?

Reply:
I think it can be fixed if we call `completeAction` on the frontend just to invalidate the local cache.

Reply:
We do it for join but not create.

Reply:
Solved and tested on clean installs and new users:

Before:
![Screenshot 2023-10-10 at 16 05 20](https://github.com/dailydotdev/apps/assets/554874/6becbe11-fa64-4f96-880f-d6a928cb474e)

After:
![Screenshot 2023-10-10 at 16 07 36](https://github.com/dailydotdev/apps/assets/554874/d9444b86-23b7-4e91-9cbd-0147a2df5068)